src/fides/api/schemas/custom_report.py

```diff
@@ -13,13 +13,40 @@ class ReportType(str, Enum):
     privacy_request = "privacy_request"
 
 
+class ColumnMapItem(FidesSchema):
+    """A map between column keys and custom labels."""
+
+    label: Optional[str] = Field(
+        default=None, description="The custom label for the column"
+    )
+    enabled: Optional[bool] = Field(
+        default=None, description="Whether the column is shown"
+    )
+
+
 class CustomReportConfig(FidesSchema):
     """The configuration for a custom report."""
 
     table_state: Dict[str, Any] = Field(
         default_factory=dict,
         description="Flexible dictionary storing UI-specific table state data without a fixed schema",
     )
-    column_map: Dict[str, str] = Field(
+    column_map: Dict[str, ColumnMapItem] = Field(
         default_factory=dict, description="A map between column keys and custom labels"
     )
+
+    @property
+    def columns_to_skip(self) -> Set[str]:
```

Comment:
What would happen if we tried to call the `columns_to_skip` and `custom_column_labels` properties on our old `column_map` version without `enabled` and `label`?